src/components/secondPassphraseInput/secondPassphraseInput.js

```diff
@@ -5,21 +5,18 @@ import { isValidPassphrase } from '../../utils/passphrase';
 class SecondPassphraseInput extends React.Component {
   componentDidMount() {
     if (this.props.hasSecondPassphrase) {
-      this.props.onError(undefined, '');
+      this.props.onChange('');
```

Comment:
This should be `this.props.onChange('', '');`, so that in Send dialog it doesn't show the field in red from the beginning.